octobot_tentacles_manager/api/inspector.py

```diff
@@ -65,3 +79,63 @@ def check_tentacle_version(version, name, origin_package, verbose=True) -> bool:
         if verbose:
             logger.error(f"Error when reading tentacle metadata: {e}")
     return True
+
+
+def _load_tentacle_class(tentacle_name):
+    # Lazy import of tentacles to let tentacles manager handle imports
+    try:
```

Comment:
We don't need to do the same thing for `RealTimeEvaluator` and `SocialEvaluator` ?

Reply:
We do, thanks for seeing this !